### ISSUE TITLE
Cannot read properties of undefined (reading 'config') error when running yarn start

### ISSUE THREAD

**Issue by abdurahman-ghazi** [sanity-io/sanity#3535]
**Describe the bug**

`[plugin:vite:css] Cannot read properties of undefined (reading 'config') ` on yarn start

**To Reproduce**

1. `sudo yarn global add sanity@dev-preview`
2. `sanity init`
3. `yarn start`
4. See error : `[plugin:vite:css] Cannot read properties of undefined (reading 'config') `

**Screenshots**

![image](https://user-images.githubusercontent.com/34869391/185370908-b64074ac-a23d-45de-8b32-76908d8f6a9b.png)

**Which versions of Sanity are you using?**

 ```bash
@sanity/cli (global)          3.0.0-dev-preview.15 (latest: 2.30.6)
@sanity/eslint-config-studio                 2.0.0 (up to date)
sanity                        3.0.0-dev-preview.15 (latest: 2.29.3)
```

**What operating system are you using?**
archlinux

**Which versions of Node.js / npm are you running?**
```bash
node -v  //v16.17.0
```
```bash
yarn -v  //1.22.19
```

**Comment by kmelve** [comment]
Hi! 

We are currently working on improving our workflows and follow-up on our open GitHub repository. In that work, we have decided to close most issues older than the release of Sanity Studio v3.

We value your feedback, so if this issue is still important to you and relevant for Sanity Studio v3, please search for relevant open issues. If you can’t find any, open a new one and link to relevant comments in this thread. For questions about how to do something, please post them in the [slack

**Comment by abdurahman-ghazi** [comment]
i found that the problem was happening because the project was inside a folder named in arabic ... i changed it to english and everything worked well

**Comment by kerns** [comment]
I had this same error on the latest versions of everything, but don't have a folder named in Arabic to change.

As I am using tailwind, I saw someone else suggested copying the tailwind config from the root level of my project to my sanity folder. This indeed fixes the issue. But it's a little unsettling not to understand why. And do these files need to be synced now?

FWIW I got over the grossness of worrying about the sync state of these files with the following, which I can confirm works just as well as copying the file down.

`ln -s ../tailwind.config.js tailwind.config.js`

Again, not sure why it wants this file, but I can just add it to the list of things I don't understand and keep moving. 🙏
